doc/getting_started/getting_started.rst

```diff
@@ -79,35 +79,49 @@ To build an example application follow these steps:
 
       $ source zephyr-env.sh
 
-#. Build the :ref:`hello_world` example project, enter:
+#. Generate project files for the :ref:`hello_world` example, enter:
 
    .. code-block:: console
 
       $ cd $ZEPHYR_BASE/samples/hello_world
-      $ make
+      $ mkdir build
+      $ cd build
+      $ cmake -GNinja ..
+
+The above will generate project files that can build the
+:ref:`hello_world` sample application using the default settings
+defined in the application's CMakeLists.txt. To build the application,
+enter:
+
+.. code-block:: console
+
+   $ cd $ZEPHYR_BASE/samples/hello_world/build
+   $ ninja
 
-The above invocation of make will build the :ref:`hello_world` sample application
-using the default settings defined in the application's Makefile. You can
-build for a different board by defining the variable BOARD with one of the
-supported boards, for example:
+You can build for a different board by defining the variable BOARD
+with one of the supported boards, for example:
```

Comment:
Is there place/command we can give folks to provide a list of the supported board names (such as arduino_101) in this  new cmake world?  (Today I can do a "make help" to see board names).